bundles/CoreBundle/config/pimcore/default.yaml

```
@@ -52,6 +52,11 @@ framework:
         buses:
             messenger.bus.pimcore-core:
         reset_on_message: true
+    rate_limiter:
+        reset_password:
+            policy: 'fixed_window'
+            limit: 3
+            interval: '60 minutes'
```

```suggestion
            limit: 5
            interval: '5 minutes'
```

Formerly the rule were set as 5 attempts by an ip and interval of 5 minutes
https://github.com/pimcore/pimcore/blob/25bb7f79edac52672891f421eab31d3fa906bc9a/bundles/AdminBundle/Security/BruteforceProtectionHandler.php#L116-L122